spec/features/user_logout_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'user logout', type: :feature do
+  let(:user) { User.create(username: "Tom", password: "password") }
+
+  it 'existing user can logout' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.